public/index.html

```diff
@@ -3,19 +3,18 @@
     <head>
         <meta charset="utf-8">
         <title>Chat</title>
+        <!-- external libraries !-->
         <script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.5.8/angular.min.js"></script>
```

Comment:
In a separate pull request, you may as well bundle your libraries as well! You can either do this as a separate bundle, or just have a single one for the entire app.